site/source/docs/api_reference/module.rst

```diff
@@ -81,6 +81,14 @@ The following ``Module`` attributes affect code execution.
 
 	If ``noExitRuntime`` is set to ``true``, the runtime is not shut down after ``run`` completes. Shutting down the runtime calls shutdown callbacks, for example ``atexit`` calls. If you want to continue using the code after ``run()`` finishes, it is necessary to set this. This is automatically set for you if you use an API command that implies that you want the runtime to not be shut down, for example ``emscripten_set_main_loop``.
 
+.. js:attribute:: Module.onRuntimeInitialized
+
+	If set, this function is called when the runtime is fully initialized, that is, when compiled code is safe to run. (An alternative to waiting for this to be called is to wait for ``main()`` to be called.)
```

Comment:
Maybe add something about how this is also after async module instantiation has completed, since a lot of users will now be looking for exactly this event to kick off all of their code.